dev/PushNotifications/PushNotificationChannel.cpp

```diff
@@ -148,6 +182,19 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
     }
     CATCH_RETURN()
 
+    HRESULT __stdcall PushNotificationChannel::OnRawNotificationReceived(unsigned int payloadLength, _In_ byte* payload, _In_ HSTRING /*correlationVector */) noexcept
+    {
+        BOOL foregroundHandled = false;
```

Comment:
Please set this to true instead. Default behavior should be that if the dev doesn't set this to true, then we don't do protocol launch.